scala/private/macros/setup_scala_toolchain.bzl

```diff
@@ -95,3 +95,69 @@ def setup_scala_toolchain(
         toolchain_type = "@io_bazel_rules_scala//scala:toolchain_type",
         visibility = visibility,
     )
+
+_DEFAULT_DEPS = {
+    "scala_compile_classpath": {
+        "any": [
```

Comment:
Was this "any" introduced to reduce duplication? I'm afraid this assumption that labels match between scala version might not always hold, leaving "any" obsolete.

I think some duplication here is fine. I would just list all major scala versions and deps lables for each dep_id. 

imho that would look simpler. wdyt?

Reply:
Indeed, `any` is here just to avoid duplication.

My goal was to keep the change minimal (here: not changing the logic of producing the relevant labels). This deduplication is not a new concept, as it expresses the same intention as the old version in the BUILD file, e.g.:

```
_SCALA_COMPILE_CLASSPATH_DEPS = [
    "@io_bazel_rules_scala_scala_compiler",
    "@io_bazel_rules_scala_scala_library",
] + (["@io_bazel_rules_scala_scala_reflect"] if SCALA_MAJOR_VERSION.startswith("2") else [
    "@io_bazel_rules_scala_scala_interfaces",
    "@io_bazel_rules_scala_scala_tasty_core",
    "@io_bazel_rules_scala_scala_asm",
    "@io_bazel_rules_scala_scala_library_2",
])
```

translated into:

```
    "scala_compile_classpath": {
        "any": [
            "@io_bazel_rules_scala_scala_compiler",
            "@io_bazel_rules_scala_scala_library",
        ],
        "2": [
            "@io_bazel_rules_scala_scala_reflect",
        ],
        "3": [
            "@io_bazel_rules_scala_scala_interfaces",
            "@io_bazel_rules_scala_scala_tasty_core",
            "@io_bazel_rules_scala_scala_asm",
            "@io_bazel_rules_scala_scala_library_2",
        ],
    },
```

Reply:
I understand your intent. Probably it's just a matter of personal taste. Since it's private and encapsulated in a function I'm totally fine with that.